lib/common/templates/ios-swift/swift.template

```diff
@@ -18,9 +18,10 @@
 //
 <%= fileHeader({file, commentStyle: 'short'}) %>
 import UIKit
+import SwiftUI
```

Comment:
Are there any issue with including both UIKit and SwitfUI? I would assume most people use one or the other? Should we make this a configuration as well?

Reply:
No, there's no additional drawback to have them both imported. We could make it a part of the configuration, but the only benefit would to have one line less of code. Import them both also does not affect app size because they are both available in every iOS environment anyway.

Reply:
The drawback would be that this could potentially be used on macos which does not have UIKit! But it does have SwiftUI. So to be more Apple Platform agnostic it would have been better to leave this out. I have been struggling with just that now.